### DISCUSSION TITLE
GitHub Student Application Rejection: University Card Expiry Date Issue

### DISCUSSION THREAD

**Discussion by ahmedrazabaloch** [community/community#88625] [GitHub Education] [answered]
### Select Topic Area

Question

### Body

Hello everyone,

I hope this message finds you well. I recently encountered an issue while applying for the GitHub Student Developer Pack, and I would appreciate some guidance on how to proceed.

When attempting to upload my university card as part of the application process, it was rejected because the card does not display an expiry date. Unfortunately, my university card does not include this information.

Has anyone else faced a similar situation, and if so, how did you address it? I'm seeking advice on how to navigate this issue and successfully complete the application.

Your insights and suggestions would be highly valuable. Thank you in advance!

**Comment by mustafa-tariqk** [comment, marked as answer]
Hey faced something similar while back,

I did it by uploading my proof of enrolment letter.

"If you do not have a dated student ID to upload, you may upload a current transcript, letter on school letterhead, or any documentation that demonstrates your current enrollment."

https://github.blog/2019-07-30-how-to-get-the-github-student-developer-pack-without-a-student-id/

If you find this answer helpful, please mark it as an answer!
